src/components/card/card.scss

```diff
@@ -8,6 +8,38 @@ $imgHeight: 233px!important;
   border: none !important;
 }
 
+.card-blog {
+
+  .card-info-event{
```

Comment:
If this was aimed at trying to solve #102, please refer back our conversation of doing these CSS overrides from the `EventsList` component.  The `Card` should not have any implementation specific styles.  Either way, these changes should not be part of this PR (as mentioned above)